Makefile

```diff
@@ -50,16 +51,19 @@ generate: packr2
 	$(GO) mod tidy
 	$(GO) generate ./...
 
-HAS_PACKR2 := $(shell command -v packr2)
-HAS_GOBIN_IN_PATH := $(shell re='(:|^)$(CLIENT_GOPATH)/bin/?(:|$$)'; if [[ "$${PATH}" =~ $${re} ]];then echo $${GOPATH}/bin;fi)
+HAS_PACKR2 := $(shell command -v packr3 && echo "OK")
 packr2:
 ifndef HAS_PACKR2
-ifndef HAS_GOBIN_IN_PATH
-	$(error "$(CLIENT_GOPATH)/bin is not in path and packr2 is not installed. Install packr2 or add "$(CLIENT_GOPATH)/bin to your path")
-endif
-	curl -SLo /tmp/packr.tar.gz https://github.com/gobuffalo/packr/releases/download/v2.6.0/packr_2.6.0_$(CLIENT_PLATFORM)_$(CLIENT_ARCH).tar.gz
-	cd /tmp && tar -xzf /tmp/packr.tar.gz
-	install /tmp/packr2 $(CLIENT_GOPATH)/bin/
+	mkdir -p tmp/packr2
+	curl -SLo tmp/packr.tar.gz https://github.com/gobuffalo/packr/releases/download/v2.6.0/packr_2.6.0_$(CLIENT_PLATFORM)_$(CLIENT_ARCH).tar.gz
+	tar -xzf tmp/packr.tar.gz -C tmp/packr2
+	install tmp/packr2/packr2 $(CLIENT_GOPATH)/bin/
+	cp $(CLIENT_GOPATH)/bin/packr2$(FILE_EXT) cmd/
```

Comment:
Are these commands necessary if GOBIN is not in PATH?  And/or, is there another solution we can try that doesn't involve copying the binary to all of these locations (and needing to update the list in the future if/when new files use packr)?

Reply:
Good point, I am not happy with all this `cp` commands going on here.

Reply:
I'm not sure what's going on here either? It would really help if you could put in the PR comment what is the root problem you are solving. Not just "it doesn't work with gitbash" but any error messages or behavior that you are running into. That way we can help figure out the problem and if the PR addresses it. The direction the PR is going right now won't be mergeable because it's not clear what it is solving.

@simongdavies Just fixed a bug in our Makefile where we weren't checking well for if the GOPATH/bin wasn't in your PATH which caused him trouble with installing packr2 on his windows machine. https://github.com/deislabs/porter/pull/1186 Perhaps that is related and may help with whatever trouble you were running into.

Reply:
The problem is that the check for HAS_GOBIN_IN_PATH does not know how to convert Windows path to Linux path, so it can never find a match:

```
Makefile:58: *** "C:\Users\artur\go/bin is not in path and packr2 is not installed. Install packr2 or add "C:\Users\artur\go/bin to your path".  Stop.
```